### PR TITLE
C/geo shapes

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: pages/blog/geo-shapes-in-crate.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
+title: Berlin and Geo Shapes in Crate
+link: https://crate.io/blog/geo-shapes-in-crate/
+author: Claus Matzinger
+description: Crate.IO added support for ``geo_shape``
```
Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: Crate.IO added support for the ``geo_shape`` datatype.

Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: header strings are plain text, so no "``" supported

File: pages/blog/geo-shapes-in-crate.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+and similar to `within` there's two ways to call on this operation: `match` and
+`intersects`. As before, the `match` variant is faster and less precise.
+
+### Disjoint
+
+`disjoint` is the opposite of `intersects` - it only matches shapes that do
+not share any points.
+
+
+## Use Case: Berlin
```
Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: ~~Use Case~~ Example: Places in Berlin

File: pages/blog/geo-shapes-in-crate.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+As a short demonstration, we use the Google Places API to gather a few
+locations in central Berlin. After importing and indexing everything with
+Crate, we can run queries based on GIS data!
+
+### Setup
+
+We are going to use two tables, the first of which will provide information
```
Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: ..., the first of ~~which~~ them ...

File: pages/blog/geo-shapes-in-crate.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+on 60 different localities (gathered from [Google's Places API][2]):
+
+```sql
+CREATE TABLE geo_shapes.places (
```
Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: I would not use the concept of schemas in this post unless there is a good reason.

File: pages/blog/geo-shapes-in-crate.html [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+  ...
+  13.515443690349803 52.40179020052256, 13.515486605693416 52.401633099079156 )')
+    using intersects
+order by 1;
+```
+By using `intersects` as a match option, we get all districts that share a
+location with the wall. Interestingly, exactly half of them have been affected:
+
+```
+
```
Comment (chaudum) [inline pages/blog/geo-shapes-in-crate.html]: remove empty line

**Comment by chaudum** [comment]
great post in general :)

File: pages/blog/geo-shapes-in-crate.html (hunk 4 above)
Comment (ChrisChinchilla) [inline pages/blog/geo-shapes-in-crate.html]: @celaus I'm OK with this, but are we missing a step that imports data? Even just a small set.

**Comment by ChrisChinchilla** [comment]
@celaus Did you see my comments questions from last week?

File: pages/blog/geo-shapes-in-crate.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+```sql
+CREATE TABLE geo_shapes.places (
+  "types" array(string),
+  "name" string,
+  "geometry" geo_point
+);
+```
+
+The second table will store the geographical information about Berlin's 12 districts:
```
Comment (celaus) [inline pages/blog/geo-shapes-in-crate.html]: add gist & insert for this data

**Comment by chaudum** [comment]
don't forget to rebase your branch. the directory structure changed!

**Pull request merged**